modules/sdk/project-templates/project-templates-war-cdi-bean-portlet/src/main/resources/archetype-resources/build.gradle

```diff
@@ -0,0 +1,57 @@
+buildscript {
+	dependencies {
+		classpath group: "com.liferay", name: "com.liferay.css.builder", version: "@com.liferay.css.builder.version@"
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.css.builder", version: "@com.liferay.gradle.plugins.css.builder.version@"
+	}
+
+	repositories {
+		maven {
+			url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+		}
+	}
+}
+
+apply plugin: "com.liferay.css.builder"
+#if (${projectType} != "workspace")
+apply plugin: "war"
+#end
+
+dependencies {
+#if (${dependencyManagementEnabled} == "true")
+	compile group: "com.liferay", name: "com.liferay.bean.portlet.api"
+	compile group: "com.liferay", name: "com.liferay.bean.portlet.cdi.extension"
+	compile group: "org.jboss.weld.servlet", name: "weld-servlet"
```

Comment:
Please add a new line between different dependency configurations (i.e. `compile` and `compileOnly`).